pom.xml

```diff
@@ -11,8 +11,8 @@
     <properties>
         <build.number>DEV</build.number>
         <jda.version>4.0.0_39</jda.version>
-        <lavaplayer.version>1.3.47</lavaplayer.version>
-        <lavalink.client.version>f9ab37f</lavalink.client.version>
+        <lavaplayer.version>1.3.50</lavaplayer.version>
+        <lavalink.client.version>4.0</lavalink.client.version>
```

Comment:
This isn't the latest version of Lavalink, this build is from Aug 2019.

`41e1025` is the latest

Reply:
It the latest tag. do we want to update to the latest commit?